### ISSUE TITLE
Intermittent WPT reftest failure in /2dcontext/shadows/canvas_shadows_002.htm

### ISSUE THREAD

**Issue by jdm** [servo/servo#5207]
From http://build.servo.org/builders/linux2/builds/998/steps/shell_1/logs/stdio:

```
/2dcontext/shadows/canvas_shadows_002.htm==/2dcontext/shadows/canvas_shadows_002-ref.htm
----------------------------------------------------------------------------------------
FAIL [Parent]
```

**Comment by Ms2ger** [comment]
This might be fixed by #9754. @mbrubeck, wdyt?

**Comment by mbrubeck** [comment]
> This might be fixed by #9754. @mbrubeck, wdyt?

The fact that it was failing before #9208 makes me think it's unrelated to #9754. But it might be worth giving it another chance anyway.

**Comment by mrobinson** [comment]
This hasn't flaked in a long time AFAICT.
